tests/functional/spec/features/routing/checkbox_count_routing.spec.js

```diff
@@ -0,0 +1,43 @@
+import ToppingCheckboxPage from "../../../generated_pages/new_routing_checkbox_count/topping-checkbox.page.js";
```

Comment:
`_routing` can be dropped from the name since it is in the `routing` folder.

Reply:
Taking `_routing` away makes it not conform with other schemas using the same naming convention. I think this is something we should deal with in the mop up card

Reply:
I only see 2 spec files with `_routing` one has already been dropped in my PR, and the other could be dropped in Ash's PR given he is making changes there. Given there is only one outstanding, I think it makes sense to update.